apps/editing-toolkit/editing-toolkit-plugin/wpcom-universal-themes/index.php

```diff
@@ -124,7 +124,7 @@ function unload_core_fse() {
 	remove_action( 'admin_menu', __NAMESPACE__ . '\hide_nav_menus_submenu' );
 
 	// Shows the AMP menu.
-	add_action( 'admin_menu', 'amp_add_customizer_link' );
+	add_action( 'admin_menu', 'amp_add_customizer_link', 11 );
```

Comment:
Same as in the previous PR, we need to check if this function exists before calling it.

Reply:
👍 

I added the guard